styles/modules/modals/_loading.scss

```diff
@@ -14,6 +14,8 @@
 
     .loadingText {
       @include center;
+      width: 100%;
```

Comment:
Why are these two style additions necessary, the mixin already centers the text vertically and horizontally?